src/main/java/org/commcare/suite/model/Detail.java

```diff
@@ -465,8 +465,8 @@ private boolean templatePathValid(String templatePathProvided) {
             return true;
         } else if (templatePathProvided != null) {
             try {
-                this.derivedPrintTemplatePath =
-                        ReferenceManager.instance().DeriveReference(templatePathProvided).getLocalURI();
+                ReferenceManager.instance().DeriveReference(templatePathProvided).getLocalURI();
```

Comment:
Does this still need to be here? Is it to surface errors, if there are any at this point?